eip/EIPS/eip-4337.md

```diff
@@ -308,7 +308,7 @@ While simulating `userOp` validation, the client should make sure that:
    3. destination address must have code (EXTCODESIZE>0)
    4. cannot call EntryPoint's methods, except `depositFor` (to avoid recursion)
```

Comment:
1. the method is `depositTo`, not depositFor
2. allowed to access the fallback (empty calldata), which calls depositTo(msg.sender)